examples/research_projects/wav2vec2/run_training.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional, Union
+
+import datasets
+import numpy as np
+import torch
+import torch.nn as nn
+from packaging import version
+
+import soundfile as sf
+from jiwer import wer
```

Comment:
Should we have a `requirements.txt` in that research project?

Reply:
think We should add this metric to `datasets` now, as we have two speech models and many speech datasets coming to the datasets lib

Reply:
Actually it's already in the dataset, will use it instead